accPlot.py

```diff
@@ -61,14 +62,21 @@ def main():
                             image (default : %(default)s)""")
 
     # check input is ok
-    if len(sys.argv) < 3:
-        msg = "\nInvalid input, please enter at least 2 parameters, e.g."
-        msg += "\npython accPlot.py timeSeries.csv.gz plot.png \n"
+    if len(sys.argv) < 2:
+        msg = "\nInvalid input, please enter at least 1 parameter, e.g."
+        msg += "\npython accPlot.py timeSeries.csv.gz \n"
         accUtils.toScreen(msg)
         parser.print_help()
         sys.exit(-1)
     args = parser.parse_args()
 
+    # determine output file name
+    if args.plotFile == None:
```

Comment:
Can we change to `if args.plotFile is None:` which is more pythonic?

Reply:
Many thanks @chanshing - I've now updated this based on your helpful suggestion